src/mono/mono/mini/llvm-jit.cpp

```diff
@@ -16,7 +16,7 @@
 #include "mini-runtime.h"
 #include "llvm-jit.h"
 
-#if defined(MONO_ARCH_LLVM_JIT_SUPPORTED) && !defined(MONO_CROSS_COMPILE)
+#if defined(MONO_ARCH_LLVM_JIT_SUPPORTED) && !defined(MONO_CROSS_COMPILE) && LLVM_API_VERSION < 1300
```

Comment:
do we have a tracking issue to remove this check once jit support is enabled ?

Reply:
This is just some very preliminary work, we don't even llvm 13 packages etc. yet.